www/views/compare.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>HeatpumpMonitor.org</title>
+<script src="https://code.jquery.com/jquery-3.6.3.min.js"></script>
+  <link rel="stylesheet" href="views/style.css?v=8" />
+  <link href="https://openenergymonitor.org/homepage/theme/favicon.ico" rel="shortcut icon">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/fontawesome.min.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/solid.min.css">
+  <script src="https://cdn.plot.ly/plotly-2.18.1.min.js" charset="utf-8"></script>
+</head>
+
+<body>
+
+<div id="header">
+  <div id="title"><span class="big">
+    <b>HeatpumpMonitor</b>.org</span></div>
+  <div id="navigation">
+    <a href="." title="Home"><i aria-hidden="true" class="fas fa-home"></i></a>
+    <a href="stats.html" title="30 Day Stats"><i aria-hidden="true" class="fas fa-table"></i></a>
+    <a href="costs.html" title="Running Costs"><i aria-hidden="true" class="fas fa-coins"></i></a>
+    <a href="graph.html" title="Comparison Charts" class="active"><i aria-hidden="true" class="fas fa-chart-line"></i></a>
+  </div>
+  <div id="tagline">An open source initiative to share and compare heat pump performance data.</div>
+</div>
+  
+<div id="app"></div>
+
+  <p>Compare hourly performance data from two systems over the last 30 days</p>
+
+  <label style="color:#0000ff">Blue</label>
+  <select id="blue_system_select">
+  </select>
+
+  <label style="color:#ff0000">Red</label>
+  <select id="red_system_select">
+  </select>
+
+  <!--
+  <label>Start</label>
+  <input type="date" value="2023-01-01">
+  <label>End</label>
+  <input type="date" value="2023-02-01">
+  
+  <label>Interval</label>
+  <select>
+    <option>Hourly</option>
+  </select>
+  -->
+  
+  
+  <div id="gd" style="height:650px"></div>
+  
+
+<div class="footer">An <a href="https://openenergymonitor.org/">OpenEnergyMonitor.org</a> community initiative</div>
+</body>
+</html>
+
+
+<script>
```

Comment:
Should we move this script into its own file?

Reply:
sure will do that